_data/talks.yml

```diff
@@ -10,125 +9,109 @@
 
 - title: "OSCAR"
   author: "The OSCAR Team"
-  date: "February 18, 2019"
-  date_sortable: 20180925
+  date: "2019-02-18"
   conference: Retreat TRR 195
   conference_url: https://www.computeralgebra.de/sfb/events/retreat-2019-of-the-sfb-trr-195/
   location: "Bad Münster am Stein, Germany"
   pdf: "BMS-talk.pdf"
 
 - title: "OSCAR - the work"
   author: "Reimer Behrends, Thomas Breuer, Sebastian Gutsche, William Hart"
-  date: "September 25, 2018"
-  date_sortable: 20180925
+  date: "2018-09-25"
   conference: Annual meeting TRR 195
   conference_url: http://www.math.uni-tuebingen.de/arbeitsbereiche/geometrie/annual-meeting-sfb-trr-195-1
   location: "Tübingen, Germany"
   pdf: "tuebingen2018-thework.pdf"
 
 - title: "OSCAR - the dream"
   author: "Mohamed Barakat, Wolfram Decker, Claus Fieker, Michael Joswig, Frank Lübeck"
-  date: "September 24, 2018"
-  date_sortable: 20180924
+  date: "2018-09-24"
   conference: Annual meeting TRR 195
   conference_url: http://www.math.uni-tuebingen.de/arbeitsbereiche/geometrie/annual-meeting-sfb-trr-195-1
   location: "Tübingen, Germany"
   pdf: "tuebingen2018-thedream.pdf"
 
 - title: "Integrating GAP and Julia"
   author: "Sebastian Gutsche"
-  date: "July 24, 2018"
-  date_sortable: 20180724
+  date: "2018-07-24"
   conference: ICMS 2018
   conference_url: http://icms-conference.org/2018/
   location: "Notre Dame, USA"
   pdf_url: "https://sebasguts.github.io/downloads/ICMS2018.pdf"
 
 - title: "OSCAR: a visionary, new computer algebra system"
   author: "Wolfram Decker, Sebastian Gutsche, William Hart, Michael Joswig"
-  date: "December 11, 2017"
-  date_sortable: 20171211
+  date: "2017-12-11"
   conference: Open Source Computer Algebra Research (OSCAR)
   conference_url: https://www.mis.mpg.de/calendar/conferences/2017/oscar2017.html
   location: "Leipzig, Germany"
   pdf: "Leipzig2017.pdf"
 
 - title: "OSCAR: a visionary, new computer algebra system"
   author: "Reimer Behrends, Thomas Breuer, Sebastian Gutsche, William Hart"
-  date: "September 27, 2017"
-  date_sortable: 20170927
+  date: "2017-09-27"
   conference: First annual meeting of the DFG collaborative research center SFB-TRR 195
   conference_url: http://www.math.rwth-aachen.de/~Sebastian.Schoennenbeck/Jahrestagung/index.html
   location: Aachen, Germany
   pdf: "Aachen2017.pdf"
 
 - title: "Nemo and Hecke: number theory in KL"
   author: "Claus Fieker"
-  date: "June 30, 2017"
-  date_sortable: 20170630
+  date: "2017-06-30"
   location: Oldenburg
 
 - title: "Nemo/Hecke: computer algebra and number theory packages for the Julia programming language"
   author: "Claus Fieker, William Hart, Tommy Hofmann, Fredrik Johansson"
-  date: "July 28, 2017"
-  date_sortable: 20170728
+  date: "2017-07-28"
   location: Kaiserslautern
 
 - title: "Basic arithmetic in Flint and Nemo"
   author: "William Hart"
-  date: "October 25, 2017"
+  date: "2017-10-23"
   location: Lyon
-  pdf: "RIAM-talk.pdf"
+  pdf: "RAIM-talk.pdf"
```

Comment:
This talk had no `date_sortable` and thus did not appear on the talks page, which is probably why nobody noticed the broken link in it. The new `date` is based on what the PDF says.